### DISCUSSION TITLE
运行单次更新后如何自动退出?

### DISCUSSION THREAD

**Discussion by pedoc** [jeessy2/ddns-go#1161] [Q&A] [answered]
目前运行ddns-go命令后不会自动退出导致命令长时间执行
实际上我只需要在开机的时候通过计划任务执行一次更新即可,如果命令不自动退出则导致计划任务长时间挂起

请问是否有参数或者配置运行执行后更新一次dns然后就自动结束退出的能力?

查找了issue和discussion,没发现或没找到类似的需求
感谢指点

**Comment by WaterLemons2k** [comment]
Windows 的任务计划程序里有一个叫做 `如果任务运行时间超过以下时间，停止任务` 的选项，试一下

**Reply by pedoc** [reply]
感谢建议,这个应该可以实现(稍微有点hack),期望程序本身能提供这种能力

**Reply by jeessy2** [reply, marked as answer]
啊？程序本身就是需要长期运行，建议你fork

**Reply by pedoc** [reply]
了解了,感谢

**Reply by pedoc** [reply]
@jeessy2 顺便问下接受PR吗,我可以添加一个参数来控制这种行为

**Reply by jeessy2** [reply]
大部分人估计用不了，暂不考虑，自行食用
